src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -4108,6 +4108,8 @@ subroutine init_atm_case_gfs(block, mesh, nCells, nEdges, nVertLevels, fg, state
       integer :: nfglevels_actual
       integer, pointer :: index_qv, index_qc, index_qr, index_qi, index_qs, index_qg, index_ni, index_nc, index_nr, index_ns, index_ng
       integer, pointer :: index_nwfa, index_nifa
+      integer, pointer :: index_MASSDEN
```

Comment:
just nit picky: can you keep the upper/lower case consistent with the neighboring code?

Reply:
Sure, will do. Thanks!

Reply:
I should mark it as resolved. Thanks!